packages/theme-ui/test/color-modes.js

```diff
@@ -518,3 +518,41 @@ test('raw color values are passed to theme-ui context when custom properties are
   )
   expect(color).toBe('tomato')
 })
+
+test('warns when localStorage is disabled', () => {
+  const originalWindow = window
+
+  const mockWindow = new Proxy(window, {
```

Comment:
What's the reason for using this instead of `jest.fn()`?

Reply:
Indeed, Object.defineProperty with a custom descriptor is enough. No need for Proxy.
Fixed.